pyproject.toml

```diff
@@ -29,7 +29,7 @@ classifiers = [
 sekoia-automation = 'sekoia_automation.cli:app'
 
 [tool.poetry.dependencies]
-python = ">=3.10,<3.12"
+python = ">=3.11,<3.13"
```

Comment:
IMHO, we should keep support for 3.10. We still have some modules that are using Python 3.10:

https://github.com/SEKOIA-IO/automation-library/blob/4ffe20a746002ce7007a2ed962c7aa3c87293921/Nybble/Dockerfile#L1